esp32s3/svd/patches/esp32s3.yaml

```diff
@@ -80,6 +80,20 @@ TIMG0:
 PCNT:
   _include: ../../../common_patches/pcnt.yaml
 
+SPI0:
+  MISC:
+    _add:
+      CS1_DIS:
```

Comment:
Should this be an array?

Reply:
Oops, agree on that, will do the fix, thanks! <3